qt/icons/Faenza.qrc

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE RCC>
 <RCC version="1.0">
-  <qresource prefix="/icons/Faenza">
+  <qresource prefix="../icons/Faenza">
```

Comment:
@mikedld  Is it simple enough as changing this to have all the resources resolve? I couldn't figure out where this path was relative to. I assumed it was Transmission/qt, but maybe it's relative to the install location or build directory.

If this is wrong, I think I'll just move all of Faenza back as it was since I'm not sure how this works and don't know how to verify. Thanks so much for all the help and time explaining things! I really do appreciate it!

Reply:
`prefix` is the [virtual] base directory the resources are scoped to when referred to via `:...` paths within code. See https://doc.qt.io/qt-6/resources.html for more info. The only real filesystem paths are those within the `file` tags, and per the aforementioned page they are "resolved relative to the directory containing the .qrc file."

Reply:
Interesting. Thanks for the documentation link! So if I understand correctly, I want the prefix to stay what it was so that all the references in code resolve normally, but I need to change the paths for each <file> entry to point to the actual location of the file, relative to where the .qrc file is.

Is that accurate? If so, I'll make those changes shortly.

Reply:
I don't think it'll work correctly w/o aliases, i.e. the final paths in the resources file are currently incorrect. Opening Faenza.qrc in Qt Designer I see paths like ":/icons/Faenza/icons/Faenza/index.theme" instead of ":/icons/Faenza/index.theme", and Qt expects it to be the latter for icons to be found, at least based on how it's currently set up — see https://doc.qt.io/qt-6/qicon.html#themeSearchPaths (specifically the mention of ":\\icons") and https://github.com/transmission/transmission/blob/c21ee87eea24eadaf4acd3eb58c837bad223018d/qt/Application.cc#L132

Reply:
Ah, thank you so much for double checking that. I decided to revert the Faenza changes I made because, as you had mentioned, it's not specific to the fix I'm putting in and I feel bad making changes that I can't personally verify before I submit for review. I don't really understand how Qt works and I really need to invest the time and present better quality work for yourself or anyone else to review.

I want to be clear that I've experienced nothing but encouragement and kind words by everyone I've interacted with in this MR, and motivations I presented above are just self presented. Thank you very much for the time you've taken to offer reviews and documentation. I learned quite a bit and I am very thankful for the help!

Let me know if I missed anything. It's been a few months so my memory on these changes isn't as clear, but I think I reset the Faenza items back to before I changed them.

Thanks again!

Reply:
Going to mark this as resolved for now. Re-open or make another comment if my assumption that this is resolved was incorrect.